### ISSUE TITLE
Add 'Widget empty' property to DataTable2 and PaginatedDataTable2 to allow showing placeholder widget when there's no data

### ISSUE THREAD

**Issue by vinnchan** [maxim-saplin/data_table_2#11]
Hi, how can I display empty result where there is no item to display

**Comment by maxim-saplin** [comment]
Hi, you can return SizedBox() or empty text:
`DataCell(Text(''))`
`DataCell(SizedBox())`

**Comment by vinnchan** [comment]
Hi, how you return for this part of codes if _items.length == 0
```
 rows: List<DataRow>.generate(_items.length, (index) {
        final Dessert dessert = _items[index];
```

**Comment by maxim-saplin** [comment]
If _items.length is zero I'd return text (e.g. Center(child: Text('No data')) ) instead of DataTable2 widget

**Comment by vinnchan** [comment]
Hi, can we display 'no item' text below table headers ?

**Comment by maxim-saplin** [comment]
Widget doesn't have the capability now, you can work around with Stack widget. Will look into adding the capabiliy.

**Comment by maxim-saplin** [comment]
Just published 2.0.4 version with that feature added
